packages/combobox-react/src/Combobox.tsx

```diff
@@ -331,7 +331,12 @@ export const Combobox: FC<ComboboxProps> = ({
             errorLabel={errorLabel}
             density={density}
             render={(inputProps) => (
-                <div className="jkl-combobox__wrapper" style={{ width }}>
+                <div
+                    className="jkl-combobox__wrapper"
+                    style={{ width }}
+                    onClick={() => setShowMenu(true)}
+                    aria-hidden="true"
```

Comment:
```suggestion
                    aria-hidden="true"
```

`aria-hidden` ødelegger for skjermleser (VoiceOver blir hvertfall ganske forvirret i Safari og Firefox), så den kan vi ikke ha her. Vi har ikke noe CSS som baserer seg på denne fra hva jeg kan se, så det skal gå fint.